src/ResourceManager/TrafficManager/AzureRM.TrafficManager.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.7.0'
+ModuleVersion = '3.0.0'
```

Comment:
@vladca please revert this change - we will be doing the version bumps as a part of the release process